src/Microsoft.CodeQuality.Analyzers/UnitTests/Performance/DoNotUseCountWhenAnyCanBeUsedTests.Fixer.cs

```diff
@@ -0,0 +1,252 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Operations;
+using Xunit;
+using static Microsoft.CodeQuality.Analyzers.Performance.UnitTests.DoNotUseCountWhenAnyCanBeUsedTestData;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.CodeQuality.Analyzers.Performance.DoNotUseCountWhenAnyCanBeUsedAnalyzer,
+    Microsoft.CodeQuality.CSharp.Analyzers.Performance.CSharpDoNotUseCountWhenAnyCanBeUsedFixer>;
+using VerifyVB = Test.Utilities.VisualBasicCodeFixVerifier<
+    Microsoft.CodeQuality.Analyzers.Performance.DoNotUseCountWhenAnyCanBeUsedAnalyzer,
+    Microsoft.CodeQuality.VisualBasic.Analyzers.Performance.BasicDoNotUseCountWhenAnyCanBeUsedFixer>;
+
+namespace Microsoft.CodeQuality.Analyzers.Performance.UnitTests
+{
+    public static class DoNotUseCountWhenAnyCanBeUsedFixerTests
```

Comment:
💡 Code fix tests do not need to be separated into their own file. All analyzer tests using the new test harness are also code fix tests, so there is no real distinguishing between the two.